src/SmartComponents/SystemAdvisor/SystemAdvisor.js

```diff
@@ -184,7 +184,8 @@ const BaseSystemAdvisor = () => {
 
       const match = rows.find((row) => row?.rule?.rule_id === rule.rule_id);
       const selected = match?.selected;
-      const isOpen = match?.isOpen || false;
+      const isOpen = match?.isOpen || false || key === 0;
```

Comment:
I think we can remove hard coded `false`
```suggestion
      const isOpen = match?.isOpen || key === 0;
```

Reply:
When going back to implement this and test, I realized that it was opening the first item every time you selected something. In the next commit Im passing in an argument 'firstLoad' so that this will only happen on first initial load